### ISSUE TITLE
Prepare a summary table of quality metrics

### ISSUE THREAD

**Issue by alexdauenhauer** [tyu0912/MIDS_W207_facial_recognition#19]
L2 distance, L1 distance. The `History()` callback in keras provides the loss at each epoch so this would be a good place to start to monitor how our model is doing

**Comment by tyu0912** [comment]
Accuracy, RMSE, Time, epochs, etc.

**Comment by mpaluta** [comment]
Score on training data
